tools/tests/test_operations.py

```diff
@@ -83,6 +93,8 @@ def test_add_gtfs_schedule_source(self, mock_catalog):
         test_municipality = "test_municipality"
         test_direct_download_url = "test_direct_download_url"
         test_license_url = "test_license_url"
+        test_status = "active"
+        test_feature = ["flex"]
```

Comment:
For consistency with what we are testing, an array of `features`.
```suggestion
        test_features = ["flex"]
```

Reply:
fixed